src/styles/blocks/main.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $widthBlock;
+
+  &__day {
+    width: $widthDay;
+    height: $heihtDay;
+    border: 1px solid black;
+    background-color: #eee;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      transition-duration: 0.5s;
+      cursor: pointer;
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day:nth-child(n + #{$i})::before {
+      content: '#{$i}';
+      font-family: sans-serif;
+      font-size: 30px;
+      height: 100%;
+      @include flexRules;
```

Comment:
```suggestion

```
Consider using this styles in the general class instead of generating them for each 31 element